### ISSUE TITLE
0.49rc1 fail 1 test on windows

### ISSUE THREAD

**Issue by sklam** [numba/numba#5448]
```pytb
ERROR: test_wrapper_address_protocol_libm (numba.tests.test_function_type.TestFunctionTypeExtensions)
Call cos and sinf from standard math library.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\ci\32\Miniconda3\envs\testenv_fbf4b59a-0935-4a9d-8efb-05e471655d84\lib\site-packages\numba\tests\test_function_type.py", line 617, in test_wrapper_address_protocol_libm
    mysin = LibM('sinf')
  File "C:\Users\ci\32\Miniconda3\envs\testenv_fbf4b59a-0935-4a9d-8efb-05e471655d84\lib\site-packages\numba\tests\test_function_type.py", line 598, in __init__
    addr = ctypes.cast(self.lib.sinf, ctypes.c_voidp).value
  File "C:\Users\ci\32\Miniconda3\envs\testenv_fbf4b59a-0935-4a9d-8efb-05e471655d84\lib\ctypes\__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "C:\Users\ci\32\Miniconda3\envs\testenv_fbf4b59a-0935-4a9d-8efb-05e471655d84\lib\ctypes\__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: function 'sinf' not found

----------------------------------------------------------------------
```

**Comment by sklam** [comment]
We'll need to skip/fix this test before 0.49 release

**Comment by seibert** [comment]
note this is failing on win-32.  It looks like win-64 is fine.

**Comment by pearu** [comment]
```
winedump -j export msvcrt.dll
```
shows that `sinf` is available only in 64-bit DLL. The 32-bit DLL contains only `sin`.

Suggest skipping the `sinf` test on 32-bit Windows systems but keep the `cos` test.

**Comment by pearu** [comment]
It is possible that on the 32-bit Windows `sin` is for 32-bit float inputs. If this is the case, then changing conditionally `self.lib.sinf` to `self.lib.sin` might work. Also, the `cos` test requires adjustment as well on the 32-bit Windows platform.

**Comment by pearu** [comment]
@sklam please see a possible fix in #5459. I cannot verify the fix myself as I don't have immediate access to the 32-bit Windows platform.
